### DISCUSSION TITLE
Multi-cluster setup service failure

### DISCUSSION THREAD

**Discussion by kotyara85** [linkerd/linkerd2#5503] [General]
Hey there,
We have multiple clusters running the same application I want to utilize linkerd to forward the traffic to another cluster if svc of a specific application is not responding.
Is linkerd capable to do that?
Thank you!

**Comment by olix0r** [comment]
This isn't implemeted out-of-the-box, but it would be fairly straightforward to implement an operator that updates a `TrafficSplit` based on the service's readiness.

**Reply by kotyara85** [reply]
@olix0r do you know of any other meshes which have this functionality out of the box?
Thank you
